networking/change-block-size.md

```diff
@@ -0,0 +1,303 @@
+---
+title: Change IP pool block size
+description: Expand or shrink the IP pool block size to efficiently manage IP pool addresses. 
+---
+
+### Big picture
+
+Change the IP pool block size to efficiently manage IP pool addresses. 
+
+### Value
+
+Because the IP pool block size is configured during install and cannot be edited, you must follow the steps outlined in this article to minimize pod connectivity disruption. 
+
+### Features
+
+This how-to guide uses the following {{site.prodname}} features:
+
+- **IPPool** resource with `blockSize` field
+
+### Concepts
+
+#### Best practices
+
+- **Create IP pools within the same cluster CIDR** 
+
+   Kubernetes expects that all pods have IP addresses within the same cluster CIDR. Although {{site.prodname}} technically supports using multiple disjoint IP pool CIDRs, we do not recommend it; pods allocated addresses outside of the Kubernetes cluster CIDR will lose network connectivity. 
+  
+- **Determine IP pool block size before installing {{site.prodname}}** 
+
+   Because the `blockSize` field cannot be edited directly after installation, you must use a three-step process after installation to change `blockSize` to mimimize disruptions to pod connectivity.
+
+### Before you begin...
+
+**Required**
+
+- Verify that you are using {{site.prodname}} IPAM. This guide is only relevant if you are using Calico IPAM.
+
+  ssh to one of your Kubernetes nodes and view the CNI configuration.  
+
+    ```bash
+    cat /etc/cni/net.d/10-calico.conflist
+     ```
+  Look for the "type" entry:
+  <pre>
+     "ipam": {
+           "type": "calico-ipam"
+      }, 
+  </pre>
+
+  If the type is “calico-ipam”, you are good to go. If the IPAM is set to something else, or the 10-calico.conflist file does not exist, you cannot use this feature in your cluster. 
+
+- Verify orchestrator support for changing the pod network CIDR
+
+  Although Kubernetes supports changing the pod network CIDR, not all orchestrators do. For example, **OpenShift** does not support this feature as described in [`osm_cluster_network_cidr configuration`](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-cluster-variables). Check your orchestrator documentation to verify. 
```

Comment:
This link is a 404 now. 

Also, I don't think changing the pod CIDR has anything to do with this, right? We're not changing the CIDR, we're just changing the block size. So we probably don't need to ask users to check this at all!

Reply:
I can't recall where that came from. Deleted.